data-oriented-programming/model-tabular-data/main.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/io;
+
+// Define a type for tabular data
+type Employee record {|
+    readonly int id;
+    string name;
```

Comment:
Hmmm, what we had initially was that. But since java hashtable compound key support is not direct(will have to update the hash method to handle the compound fields) thought of having a compound key sample.
Agree with single key is much easier. wdyt?

Reply:
It's okay. Let's keep this as it is for now.